mentorship/gsod/2022/project-ideas.md

```diff
@@ -5,16 +5,84 @@ which Keptn contributors have proposed for this year.
+
+#### New example projects
+
+Create 5-6 projects that introduce key Keptn capabilities for the Keptn curious to explore. 
+We have a number of existing tutorials to use as starters or inspiration. 
+Note that many of the existing tutorials use the [sockshop project](https://github.com/keptn-demo/sockshop) as a base;
+and some updates will be needed there.
+
+Each project should include:
+
+- Files that can be downloaded from github to provide some basic configuration with an explanation of what is provided in these files (that they would have to do themselves if they wanted to implement this functionality.
+- Specification in GitHub for what needs to be installed to provide a viable environment for the project, including integrations that must be included, configurations that must be set, and so forth.
+- Excellent information about how to run the sample project and do the exercises, including context for each step, what to look for to determine success and understand the output.  The goal is to begin to educate the user about how to get the most out of Keptn
+
+#### Separate guides for roles
+
+We want to introduce separate documentation for different Keptn user roles: 
+administrators, end users and developers.
+We suggest to work together to regroup the content so that the new structure is introduced.
+
+#### New reference pages
+
+Rewrite reference pages for files to conform to the structure proposed in the [Structure of reference pages](https://docs.google.com/document/d/1rmSGUFsnYuNc_BGHGmkW-9iB3hrOYbMeA-mJmMOjGRk/edit?usp=sharing) proposal.
+We are currently creating reference pages for critical files by capturing information in existing doc files, source code README files, etc.
+This project would be to restructure and expand the information in these files to create proper reference pages.
+It might also include relocating some information in existing files to a more appropriate location.
+
+Same for other standard documentation independent from version.
+Set a standard for what information belongs in a “Concepts” article versus
+what belongs in reference pages or guides that are currently
+located in the version-specific sections of the documentation.
+Then modify the existing Concepts articles to conform to this standard and create Concepts articles for appropriate concepts that
+are discussed in existing guides or elsewhere
+(such as README files in the software repos or existing Keptn videos).
+
+Creating missing guides is not part of this project
 
 ## Alternative project ideas
 
+We also discussed a number of alternative project ideas that can be considered.
+
 ### Documentation assessment
 
 This project would assess the Keptn documentation, to help us come with a plan for improving our documentation. In particular, we're looking for someone to look at the content organization, the clarity of the language and concepts, and to make sure it's as readable as possible for both implementors and end users.
```

Comment:
```suggestion
This project would assess the Keptn documentation, to help us come up with a plan for improving our documentation. In particular, we're looking for someone to look at the content organization, the clarity of the language and concepts, and to make sure it's as readable as possible for both implementors and end-users.
```

Reply:
"come up" is not great phraseology for formal writing.  How about "to help us devise a plan for improving our documentation."?  Could be "create a plan" or "define a plan" or something else.